cpg_infra/config/config.py

```diff
@@ -205,6 +213,13 @@ class GCPAggregator(DeserializableDataclass):
     # within itself, but this is a map of all users known to the system
     users: dict[MemberKey, CPGInfrastructureUser]
 
+    # include list of plugins enabled. This is specified explicitly to control
+    # what plugins are included from dependencies, also to allow exclusion of
+    # plugins for testing purposes
```

Comment:
Can you add a small comment just noting they're exposed through entrypoints, and to check <file> for more documentation?